src/karma.config.js

```diff
@@ -93,7 +89,6 @@ module.exports = function (config) {
         // Disable watching because karma-browserify handles this.
         watched: false,
 
-        // Configure `type` explicitly to avoid warning about .coffee files.
         type: 'js',
```

Comment:
Is `type` configuration necessary at all? What purpose does it serve here? We've already got pattern globs...

Reply:
At present it is no longer needed because the type will be inferred from the file extension, but it would be useful if for any reason we had JS files with a non-JS extension in future.